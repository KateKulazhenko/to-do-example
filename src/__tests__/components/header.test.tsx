import * as React from 'react';
import {shallow} from 'enzyme';

import Header from 'components/header/header';

describe ('components/header tests', () => {
  it('should render as expected ', () => {
      // Arrange
      // Act
       const component = shallow(
          <Header/>
       );
      // Assert
       expect (component).toMatchSnapshot();
  });
});
