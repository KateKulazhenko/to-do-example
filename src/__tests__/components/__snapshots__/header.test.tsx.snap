// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/header tests should render as expected  1`] = `
<header
  className="header"
>
  <h1
    className="header__text"
  >
    To Do yourself!
  </h1>
</header>
`;
